### PR TITLE
add relay

### PR DESCRIPTION
**Pull request by lackrobin**

### CODE REVIEW

File: relays.yaml [hunk 1]
```diff
@@ -57,4 +57,5 @@ relays:
   - 'wss://nostr-relay.nonce.academy'
   - 'wss://nostr.rewardsbunny.com'
   - 'wss://nostr.slothy.win'
+  - 'wss://relay.nostropolis.xyz/websocket
```
Comment (hugomd) [inline relays.yaml:60]: You're missing a single quote:
```suggestion
  - 'wss://relay.nostropolis.xyz/websocket'
```

**Comment by dskvr** [comment]
Thanks for the fix @hugomd!

File: relays.yaml (hunk 1 above)
Comment (lackrobin) [reply relays.yaml]: wow.. sorry about that. thanks for the fix

**Comment by lackrobin** [comment]
do i have to change settings on the relay? i can connect with clients, but on nostr.watch it apears as offline. ping @dskvr

**Pull request merged**